src/physics/cam/clubb_mf.F90

```diff
@@ -450,10 +545,76 @@ subroutine integrate_mf( nz,
        wmin = sigmaw * pwmin
        wmax = sigmaw * pwmax
 
+       do i=1,clubb_mf_nup
+         wlv = wmin + (wmax-wmin) / (real(clubb_mf_nup,r8)) * (real(i-1, r8))
+         wtv = wmin + (wmax-wmin) / (real(clubb_mf_nup,r8)) * real(i,r8)
+
+         upw(1,i) = 0.5_r8 * (wlv+wtv)
+         upa(1,i) = 0.5_r8 * erf( wtv/(sqrt(2._r8)*sigmaw) ) &
+                    - 0.5_r8 * erf( wlv/(sqrt(2._r8)*sigmaw) )
+
+         upmf(1,i)= rho_zm(1)*upa(1,i)*upw(1,i)
+
+         upu(1,i) = u(1)
+         upv(1,i) = v(1)
+
+         upqt(1,i)  = cwqt * upw(1,i) * sigmaqt/sigmaw
+         upthv(1,i) = cwthv * upw(1,i) * sigmathv/sigmaw
+       enddo
+
+       facqtu=1._r8
+       facthvu=1._r8
+       if (scalesrf) then

```

Comment:
What is this for?

Reply:
I understand now what this does. General comment: would it be possible to add a few more comments to the code? Throughout the code but also when the variables are defined at the beginning of the script. I understand how "boring" it is, and I'm guilty of doing the same thing (not doing it in this case)... but it really helps others to read the code faster! Thanks!

Reply:
sure. I can try to make a habit of more comments. For this one in particular, I added "(req'd for conservation if not running with zero-flux b.c.'s)". This functionality is anticipation of a future enhancement where we recognize the area of the plumes and therefore can lift our zero-flux b.c.'s assumption.